app/assets/stylesheets/content/_custom_actions.sass

```diff
@@ -35,8 +35,9 @@
   .custom-action
     display: flex
     justify-content: flex-end
-    flex: 1 1 auto
+    flex: 0 1 auto
     margin-bottom: 0.5rem
+    max-width: calc(100vw - 20px)
```

Comment:
What is this needed for? And why 20px?

Reply:
It's needed to prevent buttons to be longer than the screen's width or to be cut off. Because of this the buttons will fit the screen size and the text will be shortened by the text shortener. Actually the margin would be 30px on mobile (for 15px left and right) but because the buttons themself have a right margin of 10px we have to subtract this from the 30px.